packages/DropdownMenu/stories/examples/Showcase.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import Heading from "@paprika/heading";
+import { DropDownMenuStory } from "../DropDownMenu.stories.styles";
+import DropDownMenuExample from "./DropDownMenuExample";
```

Comment:
My Storybook fails to load, due to the second `D` being capitalized here, and the line above (but not for the actual file name).

Reply:
Hmm, yeah i'm seeing storybook not compiling it correctly for some reason. As those casing errors keep reappearing

Reply:
Should be fine now after the component rename